front/app/api/group_memberships/useAddMembership.ts

```diff
@@ -24,9 +25,10 @@ const useAddMembership = () => {
     onSuccess: async () => {
       queryClient.invalidateQueries({ queryKey: membershipsKeys.lists() });
       queryClient.invalidateQueries({ queryKey: groupsKeys.lists() });
+      queryClient.invalidateQueries({ queryKey: usersKeys.lists() });
 
       await streams.fetchAllWith({
-        apiEndpoint: [`${API_PATH}/users`, `${API_PATH}/users/me`],
+        apiEndpoint: [`${API_PATH}/users/me`],
```

Comment:
We've converted the users/me endpoint now right? So we can use invalidateQueries instead?

Reply:
No, we haven't. This is what I'm working on now.